docs/GroupDefs.hpp

```diff
@@ -48,6 +48,49 @@
 /*!
  * \defgroup ControlSystemGroup Control System
  * \brief Contains control system elements
+ *
+ * The control system manages the time-dependent mapping between frames, such as
+ * the fixed computational frame (grid frame) and the inertial frame. The
+ * time-dependent parameters of the mapping are adjusted by a feedback control
+ * system in order to follow the dynamical evolution of the black holes. In BBH
+ * simulations the map is typically a composition of maps that include
+ * translation, rotation, scaling, shape, etc.
+ * Each map under the governance of the control system has an associated
+ * time-dependent map parameter \f$\lambda(t)\f$ that is a piecewise Nth order
+ * polynomial. At regular time intervals the control system resets the Nth time
+ * derivative of \f$\lambda(t)\f$ to a new constant value, in order to minimize
+ * an error function \f$Q(t)\f$ that is specific to each map. At each reset
+ * time, the Nth derivative of \f$\lambda(t)\f$ is set to a function \f$U(t)\f$,
+ * called the control signal, that is determined by \f$Q(t)\f$ and its time
+ * derivatives and time integral. Note that \f$\lambda(t)\f$, \f$U(t)\f$, and
+ * \f$Q(t)\f$ can be vectors.
+ *
+ * The key components of the control system are:
+ * - FunctionsOfTime: each map has an associated FunctionOfTime which holds the
+ *   map parameter \f$\lambda(t)\f$ and relevant time derivatives.
+ * - ControlError: each map has an associated ControlError which computes
+ *   the error, \f$Q(t)\f$, in the mapping.
+ * - Averager: an averager can be used to average out the noise in the 'raw'
+ *   \f$Q(t)\f$ returned by the ControlError.
+ * - Controller: the map controller computes the control signal \f$U(t)\f$ from
+ *   \f$Q(t)\f$ and its time integral and time derivatives.
+ *   The control is accomplished by setting the Nth derivative of
+ *   \f$\lambda(t)\f$ to \f$U(t)\f$. Two common controllers are PID
+ *   (proportional/integral/derivative)
+ *   \f[ U(t) = a_{0}\int Q(t) dt + a_{1}Q(t) +a_{2}\frac{dQ}{dt} \f]
+ *   or
+ *   PND (proportional/N derivatives)
+ *   \f[ U(t) = \sum_{k=0}^{N} a_{k} \frac{d^kQ}{dt^k} \f]
+ *   The coefficients \f$ a_{k} \f$ in the computation of \f$U(t)\f$ are chosen
```

Comment:
Very nice documentation! One minor point: do we need to implement or even mention PND? Aren't PND controllers much worse than PID, to the point that we never use PND in SpEC anymore?

Reply:
Actually, it is the other way around -- we use PND almost everywhere in SpEC. But it looks like we still use PID in the ringdown, so it looks like we will be needing both.

Reply:
Thanks, @mgiesler! I thought Dan found PID worked better than PND, but I guess I remembered wrong. Anyway, if we're using both, I guess I'd just ask that the docs be consistent with this (no assuming PID or PND in the docs).

Reply:
We just review this and very much agree with Geoffrey, very vice documentation! Quite possibly the clearest documentation we have in the code :)